# Translations template for ironic.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the ironic project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: ironic 2014.2.dev151.g8a0923c\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-09-30 06:47+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: ironic/api/controllers/v1/driver.py:116
#: ironic/api/controllers/v1/node.py:555
msgid "Method not specified"
msgstr ""

#: ironic/api/controllers/v1/node.py:309
#, python-format
msgid "Node %(node)s is already in the '%(state)s' state."
msgstr ""

#: ironic/api/controllers/v1/node.py:322
#, python-format
msgid "Node %s is already being provisioned or decommissioned."
msgstr ""

#: ironic/api/controllers/v1/node.py:593
msgid "Chassis id not specified."
msgstr ""

#: ironic/api/controllers/v1/node.py:774
#, python-format
msgid "Node %s can not be updated while a state transition is in progress."
msgstr ""

#: ironic/api/controllers/v1/port.py:181
msgid "Node id not specified."
msgstr ""

#: ironic/api/controllers/v1/types.py:136
#, python-format
msgid "'%s' is an internal attribute and can not be updated"
msgstr ""

#: ironic/api/controllers/v1/types.py:140
#, python-format
msgid "'%s' is a mandatory attribute and can not be removed"
msgstr ""

#: ironic/api/controllers/v1/types.py:145
msgid "'add' and 'replace' operations needs value"
msgstr ""

#: ironic/api/controllers/v1/types.py:176
#, python-format
msgid "Wrong type. Expected '%(type)s', got '%(value)s'"
msgstr ""

#: ironic/api/controllers/v1/utils.py:33
msgid "Limit must be positive"
msgstr ""

#: ironic/api/controllers/v1/utils.py:40
#, python-format
msgid "Invalid sort direction: %s. Acceptable values are 'asc' or 'desc'"
msgstr ""

#: ironic/api/controllers/v1/utils.py:50
#, python-format
msgid "Adding a new attribute (%s) to the root of  the resource is not allowed"
msgstr ""

#: ironic/api/middleware/auth_token.py:41
#, python-format
msgid "Cannot compile public API routes: %s"
msgstr ""

#: ironic/api/middleware/parsable_error.py:54
#, python-format
msgid "ErrorDocumentMiddleware received an invalid status %s"
msgstr ""

#: ironic/common/disk_partitioner.py:166
#, python-format
msgid ""
"Disk partitioning failed on device %(device)s. Processes with the "
"following PIDs are holding it: %(pids)s. Time out waiting for completion."
msgstr ""

#: ironic/common/disk_partitioner.py:172
#, python-format
msgid ""
"Disk partitioning failed on device %(device)s. Fuser exited with "
"\"%(fuser_err)s\". Time out waiting for completion."
msgstr ""

#: ironic/common/exception.py:58
msgid "An unknown exception occurred."
msgstr ""

#: ironic/common/exception.py:99
msgid "Not authorized."
msgstr ""

#: ironic/common/exception.py:104
msgid "Operation not permitted."
msgstr ""

#: ironic/common/exception.py:108
msgid "Unacceptable parameters."
msgstr ""

#: ironic/common/exception.py:113
msgid "Conflict."
msgstr ""

#: ironic/common/exception.py:118
msgid "Resource temporarily unavailable, please retry."
msgstr ""

#: ironic/common/exception.py:123
msgid "Invalid resource state."
msgstr ""

#: ironic/common/exception.py:127
#, python-format
msgid "A node with UUID %(uuid)s already exists."
msgstr ""

#: ironic/common/exception.py:131
#, python-format
msgid "A port with MAC address %(mac)s already exists."
msgstr ""

#: ironic/common/exception.py:135
#, python-format
msgid "A chassis with UUID %(uuid)s already exists."
msgstr ""

#: ironic/common/exception.py:139
#, python-format
msgid "A port with UUID %(uuid)s already exists."
msgstr ""

#: ironic/common/exception.py:143
#, python-format
msgid ""
"Instance %(instance_uuid)s is already associated with a node, it cannot "
"be associated with this other node %(node)s"
msgstr ""

#: ironic/common/exception.py:148
#, python-format
msgid "Expected a uuid but received %(uuid)s."
msgstr ""

#: ironic/common/exception.py:152
#, python-format
msgid "Expected an uuid or int but received %(identity)s."
msgstr ""

#: ironic/common/exception.py:156
#, python-format
msgid "Expected a MAC address but received %(mac)s."
msgstr ""

#: ironic/common/exception.py:160
#, python-format
msgid "Invalid state '%(state)s' requested for node %(node)s."
msgstr ""

#: ironic/common/exception.py:164
#, python-format
msgid "Couldn't apply patch '%(patch)s'. Reason: %(reason)s"
msgstr ""

#: ironic/common/exception.py:168
#, python-format
msgid "Failed to deploy instance: %(reason)s"
msgstr ""

#: ironic/common/exception.py:172 ironic/common/exception.py:176
#, python-format
msgid "Image %(image_id)s is unacceptable: %(reason)s"
msgstr ""

#: ironic/common/exception.py:182 ironic/common/exception.py:186
#, python-format
msgid "%(err)s"
msgstr ""

#: ironic/common/exception.py:190
msgid "Resource could not be found."
msgstr ""

#: ironic/common/exception.py:195
#, python-format
msgid "Failed to load DHCP provider %(dhcp_provider_name)s."
msgstr ""

#: ironic/common/exception.py:199
#, python-format
msgid "Failed to load driver %(driver_name)s."
msgstr ""

#: ironic/common/exception.py:203
#, python-format
msgid "Image %(image_id)s could not be found."
msgstr ""

#: ironic/common/exception.py:207
#, python-format
msgid "No valid host was found. Reason: %(reason)s"
msgstr ""

#: ironic/common/exception.py:211
#, python-format
msgid "Instance %(instance)s could not be found."
msgstr ""

#: ironic/common/exception.py:215
#, python-format
msgid "Node %(node)s could not be found."
msgstr ""

#: ironic/common/exception.py:219
#, python-format
msgid "Node %(node)s is associated with instance %(instance)s."
msgstr ""

#: ironic/common/exception.py:223
#, python-format
msgid "Port %(port)s could not be found."
msgstr ""

#: ironic/common/exception.py:227
#, python-format
msgid "Update DHCP options on port: %(port_id)s failed."
msgstr ""

#: ironic/common/exception.py:231
#, python-format
msgid "Retrieve IP address on port: %(port_id)s failed."
msgstr ""

#: ironic/common/exception.py:235
#, python-format
msgid "Invalid IPv4 address %(ip_address)s."
msgstr ""

#: ironic/common/exception.py:239
#, python-format
msgid "Update MAC address on port: %(port_id)s failed."
msgstr ""

#: ironic/common/exception.py:243
#, python-format
msgid "Chassis %(chassis)s could not be found."
msgstr ""

#: ironic/common/exception.py:247
#, python-format
msgid "Conductor %(conductor)s could not be found."
msgstr ""

#: ironic/common/exception.py:251
#, python-format
msgid "Conductor %(conductor)s already registered."
msgstr ""

#: ironic/common/exception.py:255
#, python-format
msgid "Failed to set node power state to %(pstate)s."
msgstr ""

#: ironic/common/exception.py:259
msgid "An exclusive lock is required, but the current context has a shared lock."
msgstr ""

#: ironic/common/exception.py:264
#, python-format
msgid "Failed to toggle maintenance-mode flag for node %(node)s: %(reason)s"
msgstr ""

#: ironic/common/exception.py:269
#, python-format
msgid "Console access is not enabled on node %(node)s"
msgstr ""

#: ironic/common/exception.py:273
#, python-format
msgid ""
"The %(op)s operation can't be performed on node %(node)s because it's in "
"maintenance mode."
msgstr ""

#: ironic/common/exception.py:278
#, python-format
msgid ""
"Can not change instance association while node %(node)s is in power state"
" %(pstate)s."
msgstr ""

#: ironic/common/exception.py:283
#, python-format
msgid ""
"Cannot complete the requested action because chassis %(chassis)s contains"
" nodes."
msgstr ""

#: ironic/common/exception.py:288
#, python-format
msgid "IPMI call failed: %(cmd)s."
msgstr ""

#: ironic/common/exception.py:292
#, python-format
msgid "Failed to establish SSH connection to host %(host)s."
msgstr ""

#: ironic/common/exception.py:296
#, python-format
msgid "Failed to execute command via SSH: %(cmd)s."
msgstr ""

#: ironic/common/exception.py:300
#, python-format
msgid "Unsupported object type %(objtype)s"
msgstr ""

#: ironic/common/exception.py:304
#, python-format
msgid "Cannot call %(method)s on orphaned %(objtype)s object"
msgstr ""

#: ironic/common/exception.py:308
#, python-format
msgid "Driver %(driver)s does not support %(extension)s."
msgstr ""

#: ironic/common/exception.py:312
#, python-format
msgid "Version %(objver)s of %(objname)s is not supported"
msgstr ""

#: ironic/common/exception.py:316
#, python-format
msgid "Connection to glance host %(host)s:%(port)s failed: %(reason)s"
msgstr ""

#: ironic/common/exception.py:321
#, python-format
msgid "Not authorized for image %(image_id)s."
msgstr ""

#: ironic/common/exception.py:325
#, python-format
msgid "Invalid image href %(image_href)s."
msgstr ""

#: ironic/common/exception.py:329
msgid "Unauthorised for keystone service catalog."
msgstr ""

#: ironic/common/exception.py:337
#, python-format
msgid ""
"Service type %(service_type)s with endpoint type %(endpoint_type)s not "
"found in keystone service catalog."
msgstr ""

#: ironic/common/exception.py:342
msgid "Connection failed"
msgstr ""

#: ironic/common/exception.py:346
msgid "Requested OpenStack Images API is forbidden"
msgstr ""

#: ironic/common/exception.py:354
msgid "The provided endpoint is invalid"
msgstr ""

#: ironic/common/exception.py:358
msgid "Unable to communicate with the server."
msgstr ""

#: ironic/common/exception.py:374
#, python-format
msgid "Could not find config at %(path)s"
msgstr ""

#: ironic/common/exception.py:378
#, python-format
msgid ""
"Node %(node)s is locked by host %(host)s, please retry after the current "
"operation is completed."
msgstr ""

#: ironic/common/exception.py:383
#, python-format
msgid "Node %(node)s found not to be locked on release"
msgstr ""

#: ironic/common/exception.py:387
msgid ""
"Requested action cannot be performed due to lack of free conductor "
"workers."
msgstr ""

#: ironic/common/exception.py:397
#, python-format
msgid "Invalid configuration file. %(error_msg)s"
msgstr ""

#: ironic/common/exception.py:401
#, python-format
msgid "Driver %(driver)s could not be loaded. Reason: %(reason)s."
msgstr ""

#: ironic/common/exception.py:409
#, python-format
msgid "Could not find pid in pid file %(pid_path)s"
msgstr ""

#: ironic/common/exception.py:413
#, python-format
msgid "Console subprocess failed to start. %(error)s"
msgstr ""

#: ironic/common/exception.py:417
#, python-format
msgid "Failed to create the password file. %(error)s"
msgstr ""

#: ironic/common/exception.py:425
#, python-format
msgid "%(operation)s failed, error: %(error)s"
msgstr ""

#: ironic/common/exception.py:429
#, python-format
msgid ""
"DRAC client failed. Last error (cURL error code): %(last_error)s, fault "
"string: \"%(fault_string)s\" response_code: %(response_code)s"
msgstr ""

#: ironic/common/exception.py:436
#, python-format
msgid "DRAC %(operation)s failed. Reason: %(error)s"
msgstr ""

#: ironic/common/exception.py:440
#, python-format
msgid "DRAC failed to create a configuration job. Reason: %(error)s"
msgstr ""

#: ironic/common/exception.py:445
#, python-format
msgid ""
"Invalid filter dialect '%(invalid_filter)s'. Supported options are "
"%(supported)s"
msgstr ""

#: ironic/common/exception.py:450
#, python-format
msgid "Failed to get sensor data for node %(node)s. Error: %(error)s"
msgstr ""

#: ironic/common/exception.py:455
#, python-format
msgid "Failed to parse sensor data for node %(node)s. Error: %(error)s"
msgstr ""

#: ironic/common/exception.py:460
#, python-format
msgid ""
"Disk volume where '%(path)s' is located doesn't have enough disk space. "
"Required %(required)d MiB, only %(actual)d MiB available space present."
msgstr ""

#: ironic/common/exception.py:466
#, python-format
msgid "Creating %(image_type)s image failed: %(error)s"
msgstr ""

#: ironic/common/exception.py:470
#, python-format
msgid "Swift operation '%(operation)s' failed: %(error)s"
msgstr ""

#: ironic/common/exception.py:474
#, python-format
msgid "SNMP operation '%(operation)s' failed: %(error)s"
msgstr ""

#: ironic/common/exception.py:478
#, python-format
msgid "Failed to create a file system. File system %(fs)s is not supported."
msgstr ""

#: ironic/common/hash_ring.py:68
msgid "Invalid hosts supplied when building HashRing."
msgstr ""

#: ironic/common/hash_ring.py:81
msgid "Invalid data supplied to HashRing.get_hosts."
msgstr ""

#: ironic/common/hash_ring.py:146
#, python-format
msgid "The driver '%s' is unknown."
msgstr ""

#: ironic/common/images.py:260
msgid "'qemu-img info' parsing failed."
msgstr ""

#: ironic/common/images.py:266
#, python-format
msgid "fmt=%(fmt)s backed by: %(backing_file)s"
msgstr ""

#: ironic/common/images.py:281
#, python-format
msgid "Converted to raw, but format is now %s"
msgstr ""

#: ironic/common/keystone.py:71
msgid "Keystone API endpoint is missing"
msgstr ""

#: ironic/common/keystone.py:91
#, python-format
msgid "Could not perform authorization process for service catalog: %s"
msgstr ""

#: ironic/common/keystone.py:96
msgid "No keystone service catalog loaded"
msgstr ""

#: ironic/common/policy.py:31
msgid "JSON file representing policy."
msgstr ""

#: ironic/common/policy.py:34
msgid "Rule checked when requested rule is not found."
msgstr ""

#: ironic/common/pxe_utils.py:100
#, python-format
msgid "Failed to get IP address for any port on node %s."
msgstr ""

#: ironic/common/swift.py:97
msgid "put container"
msgstr ""

#: ironic/common/swift.py:108
msgid "put object"
msgstr ""

#: ironic/common/swift.py:128
msgid "head account"
msgstr ""

#: ironic/common/swift.py:156
msgid "delete object"
msgstr ""

#: ironic/common/swift.py:172
msgid "head object"
msgstr ""

#: ironic/common/swift.py:187
msgid "post object"
msgstr ""

#: ironic/common/utils.py:117
msgid "Invalid private key"
msgstr ""

#: ironic/common/utils.py:164
#, python-format
msgid "Invalid backend: %s"
msgstr ""

#: ironic/common/glance_service/v2/image_service.py:129
#, python-format
msgid "The given image info does not have a valid image id: %s"
msgstr ""

#: ironic/common/glance_service/v2/image_service.py:155
msgid ""
"Swift temporary URLs require a shared secret to be created. You must "
"provide \"swift_temp_url_key\" as a config option."
msgstr ""

#: ironic/common/glance_service/v2/image_service.py:159
msgid ""
"Swift temporary URLs require a Swift endpoint URL. You must provide "
"\"swift_endpoint_url\" as a config option."
msgstr ""

#: ironic/common/glance_service/v2/image_service.py:163
msgid ""
"Swift temporary URLs require a Swift account string. You must provide "
"\"swift_account\" as a config option."
msgstr ""

#: ironic/common/glance_service/v2/image_service.py:167
msgid "\"swift_temp_url_duration\" must be a positive integer."
msgstr ""

#: ironic/conductor/manager.py:253
msgid "Invalid method call: update_node can not change node state."
msgstr ""

#: ironic/conductor/manager.py:291 ironic/conductor/manager.py:433
msgid "No free conductor workers available"
msgstr ""

#: ironic/conductor/manager.py:481
#, python-format
msgid ""
"RPC do_node_deploy called to rebuild %(node)s, but provision state is "
"%(curstate)s. State must be one of : %(states)s."
msgstr ""

#: ironic/conductor/manager.py:488
#, python-format
msgid ""
"RPC do_node_deploy called for %(node)s, but provision state is already "
"%(state)s."
msgstr ""

#: ironic/conductor/manager.py:494
msgid "provisioning"
msgstr ""

#: ironic/conductor/manager.py:501
#, python-format
msgid "RPC do_node_deploy failed to validate deploy info. Error: %(msg)s"
msgstr ""

#: ironic/conductor/manager.py:532
#, python-format
msgid "Failed to deploy. Error: %s"
msgstr ""

#: ironic/conductor/manager.py:575
#, python-format
msgid "RPC do_node_tear_down not allowed for node %(node)s in state %(state)s"
msgstr ""

#: ironic/conductor/manager.py:588
#, python-format
msgid "RPC do_node_tear_down failed to validate power info. Error: %(msg)s"
msgstr ""

#: ironic/conductor/manager.py:620
#, python-format
msgid "Failed to tear down. Error: %s"
msgstr ""

#: ironic/conductor/manager.py:647
#, python-format
msgid ""
"During sync_power_state, max retries exceeded for node %(node)s, node "
"state %(actual)s does not match expected state '%(state)s'. Updating DB "
"state to '%(actual)s' Switching node to maintenance mode."
msgstr ""

#: ironic/conductor/manager.py:676
msgid "Driver returns ERROR state."
msgstr ""

#: ironic/conductor/manager.py:902
msgid "not supported"
msgstr ""

#: ironic/conductor/manager.py:931
msgid "The node is already in maintenance mode"
msgstr ""

#: ironic/conductor/manager.py:932
msgid "The node is not in maintenance mode"
msgstr ""

#: ironic/conductor/manager.py:975
#, python-format
msgid "Node %s can't be deleted because it's not powered off"
msgstr ""

#: ironic/conductor/manager.py:1047
msgid "enabled"
msgstr ""

#: ironic/conductor/manager.py:1047
msgid "disabled"
msgstr ""

#: ironic/conductor/manager.py:1067
msgid "enabling"
msgstr ""

#: ironic/conductor/manager.py:1067
msgid "disabling"
msgstr ""

#: ironic/conductor/manager.py:1068
#, python-format
msgid "Error %(op)s the console on node %(node)s. Reason: %(error)s"
msgstr ""

#: ironic/conductor/rpcapi.py:94
#, python-format
msgid "No conductor service registered which supports driver %s."
msgstr ""

#: ironic/conductor/utils.py:72 ironic/conductor/utils.py:116
#, python-format
msgid "Failed to change power state to '%(target)s'. Error: %(error)s"
msgstr ""

#: ironic/conductor/utils.py:139
#, python-format
msgid "Timeout reached while waiting for callback for node %s"
msgstr ""

#: ironic/conductor/utils.py:145
#, python-format
msgid "Cleanup failed for node %(node)s after deploy timeout:  %(error)s"
msgstr ""

#: ironic/conductor/utils.py:157
msgid ""
"Deploy timed out, but an unhandled exception was encountered while "
"aborting. More info may be found in the log file."
msgstr ""

#: ironic/db/sqlalchemy/api.py:327
msgid "Cannot overwrite UUID for an existing Node."
msgstr ""

#: ironic/db/sqlalchemy/api.py:407
msgid "Cannot overwrite UUID for an existing Port."
msgstr ""

#: ironic/db/sqlalchemy/api.py:470
msgid "Cannot overwrite UUID for an existing Chassis."
msgstr ""

#: ironic/dhcp/neutron.py:66
msgid "Neutron auth_strategy should be either \"noauth\" or \"keystone\"."
msgstr ""

#: ironic/dhcp/neutron.py:149
#, python-format
msgid ""
"No VIFs found for node %(node)s when attempting to update DHCP BOOT "
"options."
msgstr ""

#: ironic/dhcp/neutron.py:163
#, python-format
msgid "Failed to set DHCP BOOT options for any port on node %s."
msgstr ""

#: ironic/drivers/base.py:412
#, python-format
msgid "Vendor interface does not support driver vendor_passthru method: %s"
msgstr ""

#: ironic/drivers/drac.py:34 ironic/drivers/fake.py:153
msgid "Unable to import pywsman library"
msgstr ""

#: ironic/drivers/fake.py:92
msgid "Unable to import pyghmi IPMI library"
msgstr ""

#: ironic/drivers/fake.py:106 ironic/drivers/pxe.py:111
msgid "Unable to import seamicroclient library"
msgstr ""

#: ironic/drivers/fake.py:129 ironic/drivers/pxe.py:138
msgid "Unable to import iboot library"
msgstr ""

#: ironic/drivers/fake.py:141 ironic/drivers/ilo.py:43 ironic/drivers/ilo.py:66
#: ironic/drivers/pxe.py:160
msgid "Unable to import proliantutils library"
msgstr ""

#: ironic/drivers/fake.py:167 ironic/drivers/pxe.py:182
msgid "Unable to import pysnmp library"
msgstr ""

#: ironic/drivers/pxe.py:88
msgid "Unable to import pyghmi library"
msgstr ""

#: ironic/drivers/utils.py:27 ironic/drivers/modules/fake.py:36
#, python-format
msgid "Unsupported method (%s) passed through to vendor extension."
msgstr ""

#: ironic/drivers/utils.py:30 ironic/drivers/modules/fake.py:39
msgid "Method not specified when calling vendor extension."
msgstr ""

#: ironic/drivers/utils.py:206
#, python-format
msgid "Invalid boot_mode parameter '%s'."
msgstr ""

#: ironic/drivers/modules/agent.py:132
#: ironic/drivers/modules/iscsi_deploy.py:262
#, python-format
msgid ""
"Node %s failed to power off while handling deploy failure. This may be a "
"serious condition. Node should be removed from Ironic or put in "
"maintenance mode until the problem is resolved."
msgstr ""

#: ironic/drivers/modules/agent.py:233
#, python-format
msgid "Node %s failed to validate deploy image info"
msgstr ""

#: ironic/drivers/modules/agent.py:362 ironic/drivers/modules/agent.py:375
#, python-format
msgid "No handler for method %s"
msgstr ""

#: ironic/drivers/modules/agent.py:423
msgid "Node failed to get image for deploy."
msgstr ""

#: ironic/drivers/modules/agent.py:427
msgid "Node failed to move to active state."
msgstr ""

#: ironic/drivers/modules/agent.py:473
#, python-format
msgid "node %(node)s command status errored: %(error)s"
msgstr ""

#: ironic/drivers/modules/agent.py:529
#, python-format
msgid "Unknown lookup payloadversion: %s"
msgstr ""

#: ironic/drivers/modules/agent.py:561
#, python-format
msgid "Malformed network interfaces lookup: %s"
msgstr ""

#: ironic/drivers/modules/agent.py:589
#, python-format
msgid "No ports matching the given MAC addresses %sexist in the database."
msgstr ""

#: ironic/drivers/modules/agent.py:633
#, python-format
msgid ""
"Ports matching mac addresses match multiple nodes. MACs: %(macs)s. Port "
"ids: %(port_ids)s"
msgstr ""

#: ironic/drivers/modules/agent_client.py:43
msgid "Agent driver requires agent_url in driver_info"
msgstr ""

#: ironic/drivers/modules/console_utils.py:88
#, python-format
msgid ""
"Cannot create directory '%(path)s' for console PID file. Reason: "
"%(reason)s."
msgstr ""

#: ironic/drivers/modules/console_utils.py:209
#, python-format
msgid ""
"%(exec_error)s\n"
"Command: %(command)s"
msgstr ""

#: ironic/drivers/modules/console_utils.py:226
#, python-format
msgid ""
"Command: %(command)s.\n"
"Exit code: %(return_code)s.\n"
"Stdout: %(stdout)r\n"
"Stderr: %(stderr)r"
msgstr ""

#: ironic/drivers/modules/console_utils.py:237
#, python-format
msgid "Timeout while waiting for console subprocess to start for node %s."
msgstr ""

#: ironic/drivers/modules/console_utils.py:264
#, python-format
msgid "Could not stop the console for node '%(node)s'. Reason: %(err)s."
msgstr ""

#: ironic/drivers/modules/deploy_utils.py:288
#, python-format
msgid "Parent device '%s' not found"
msgstr ""

#: ironic/drivers/modules/deploy_utils.py:305
#, python-format
msgid "Root device '%s' not found"
msgstr ""

#: ironic/drivers/modules/deploy_utils.py:308
#, python-format
msgid "Swap device '%s' not found"
msgstr ""

#: ironic/drivers/modules/deploy_utils.py:312
#, python-format
msgid "Ephemeral device '%s' not found"
msgstr ""

#: ironic/drivers/modules/deploy_utils.py:405
#, python-format
msgid ""
"%(error_msg)s. The following parameters were not passed to ironic: "
"%(missing_info)s"
msgstr ""

#: ironic/drivers/modules/fake.py:57 ironic/drivers/modules/ipminative.py:338
#, python-format
msgid "set_power_state called with an invalid power state: %s."
msgstr ""

#: ironic/drivers/modules/fake.py:104
msgid "Parameter 'bar' not passed to method 'first_method'."
msgstr ""

#: ironic/drivers/modules/fake.py:132
msgid "Parameter 'bar' not passed to method 'second_method'."
msgstr ""

#: ironic/drivers/modules/fake.py:181 ironic/drivers/modules/ipminative.py:404
#: ironic/drivers/modules/ipmitool.py:694
#: ironic/drivers/modules/seamicro.py:541 ironic/drivers/modules/ssh.py:635
#: ironic/drivers/modules/ilo/deploy.py:472
#, python-format
msgid "Invalid boot device %s specified."
msgstr ""

#: ironic/drivers/modules/iboot.py:38
msgid "IP address of the node. Required."
msgstr ""

#: ironic/drivers/modules/iboot.py:39 ironic/drivers/modules/seamicro.py:69
msgid "username. Required."
msgstr ""

#: ironic/drivers/modules/iboot.py:40 ironic/drivers/modules/seamicro.py:67
msgid "password. Required."
msgstr ""

#: ironic/drivers/modules/iboot.py:43
msgid "iBoot PDU relay id; default is 1. Optional."
msgstr ""

#: ironic/drivers/modules/iboot.py:44
msgid "iBoot PDU port; default is 9100. Optional."
msgstr ""

#: ironic/drivers/modules/iboot.py:54
#, python-format
msgid "The following iBoot credentials were not supplied to iBoot PDU driver: %s."
msgstr ""

#: ironic/drivers/modules/iboot.py:66
msgid "iBoot PDU relay id must be an integer."
msgstr ""

#: ironic/drivers/modules/iboot.py:73
msgid "iBoot PDU port must be an integer."
msgstr ""

#: ironic/drivers/modules/iboot.py:108
#, python-format
msgid ""
"Cannot get power status for node '%(node)s'. iBoot get_relays() returned "
"'%(resp)s'."
msgstr ""

#: ironic/drivers/modules/iboot.py:188 ironic/drivers/modules/ipmitool.py:613
#: ironic/drivers/modules/snmp.py:647 ironic/drivers/modules/ssh.py:550
#, python-format
msgid "set_power_state called with invalid power state %s."
msgstr ""

#: ironic/drivers/modules/ipminative.py:63
msgid "IP of the node's BMC. Required."
msgstr ""

#: ironic/drivers/modules/ipminative.py:64
msgid "IPMI password. Required."
msgstr ""

#: ironic/drivers/modules/ipminative.py:65
msgid "IPMI username. Required."
msgstr ""

#: ironic/drivers/modules/ipminative.py:68
#: ironic/drivers/modules/ipmitool.py:96
msgid "node's UDP port to connect to. Only required for console access."
msgstr ""

#: ironic/drivers/modules/ipminative.py:91
#: ironic/drivers/modules/ipmitool.py:212
#, python-format
msgid "The following IPMI credentials are not supplied to IPMI driver: %s."
msgstr ""

#: ironic/drivers/modules/ipminative.py:110
#: ironic/drivers/modules/ipmitool.py:233
msgid "IPMI terminal port is not an integer."
msgstr ""

#: ironic/drivers/modules/ipminative.py:494
msgid "IPMI terminal port not supplied to the IPMI driver."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:70
msgid "IP address or hostname of the node. Required."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:73
msgid "password. Optional."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:74
#, python-format
msgid "privilege level; default is ADMINISTRATOR. One of %s. Optional."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:76
msgid "username; default is NULL user. Optional."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:77
msgid ""
"bridging_type; default is \"no\". One of \"single\", \"dual\", \"no\". "
"Optional."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:79
msgid ""
"transit channel for bridged request. Required only if ipmi_bridging is "
"set to \"dual\"."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:81
msgid ""
"transit address for bridged request. Required only if ipmi_bridging is "
"set to \"dual\"."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:83
msgid ""
"destination channel for bridged request. Required only if ipmi_bridging "
"is set to \"single\" or \"dual\"."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:86
msgid ""
"destination address for bridged request. Required only if ipmi_bridging "
"is set to \"single\" or \"dual\"."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:89
msgid ""
"local IPMB address for bridged requests. Used only if ipmi_bridging is "
"set to \"single\" or \"dual\". Optional."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:244
#, python-format
msgid ""
"Value for ipmi_bridging is provided as %s, but IPMI bridging is not "
"supported by the IPMI utility installed on host. Ensure ipmitool version "
"is > 1.8.11"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:267
#, python-format
msgid "%(param)s not provided"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:270
#, python-format
msgid ""
"Invalid value for ipmi_bridging: %(bridging_type)s, the valid value can "
"be one of: %(bridging_types)s"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:278
#, python-format
msgid ""
"Invalid privilege level value:%(priv_level)s, the valid value can be one "
"of %(valid_levels)s"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:507
#, python-format
msgid "parse ipmi sensor data failed, unknown sensor type data: %(sensors_data)s"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:545
#, python-format
msgid ""
"parse ipmi sensor data failed, get nothing with input data: "
"%(sensors_data)s"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:558
#: ironic/drivers/modules/ipmitool.py:649
#: ironic/drivers/modules/ipmitool.py:792
#: ironic/drivers/modules/ipmitool.py:927
msgid ""
"Unable to locate usable ipmitool command in the system path when checking"
" ipmitool version"
msgstr ""

#: ironic/drivers/modules/ipmitool.py:880
msgid "Parameter raw_bytes (string of bytes) was not specified."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:887
#, python-format
msgid "Unsupported method (%s) passed to IPMItool driver."
msgstr ""

#: ironic/drivers/modules/ipmitool.py:945
msgid "IPMI terminal port not supplied to IPMI driver."
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:119
msgid "Cannot validate iSCSI deploy"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:129
#, python-format
msgid ""
"Cannot validate parameter for iSCSI deploy. Invalid parameter %(param)s. "
"Reason: %(reason)s"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:135
#, python-format
msgid "'%s' is not an integer value."
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:164
#, python-format
msgid ""
"Root partition is too small for requested image. Image size: %(image_mb)d"
" MB, Root size: %(root_mb)d MB"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:219
msgid "Deploy key does not match"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:235
#, python-format
msgid "Parameters %s were not passed to ironic for deploy."
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:295
msgid "Failure in deploy ramdisk."
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:309 ironic/drivers/modules/pxe.py:487
#: ironic/drivers/modules/ilo/deploy.py:568
msgid "Failed to continue iSCSI deployment."
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:367
#, python-format
msgid "Failed to connect to Glance to get the properties of the image %s"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:371
#, python-format
msgid "Image %s not found in Glance"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:381
#, python-format
msgid "Image %(image)s is missing the following properties: %(properties)s"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:398 ironic/drivers/modules/ssh.py:497
#, python-format
msgid "Node %s does not have any port associated with it."
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:407
msgid ""
"Couldn't get the URL of the Ironic API service from the configuration "
"file or keystone catalog."
msgstr ""

#: ironic/drivers/modules/pxe.py:97
msgid "UUID (from Glance) of the deployment kernel. Required."
msgstr ""

#: ironic/drivers/modules/pxe.py:99
msgid "UUID (from Glance) of the ramdisk that is mounted at boot time. Required."
msgstr ""

#: ironic/drivers/modules/pxe.py:121
msgid "Cannot validate PXE bootloader"
msgstr ""

#: ironic/drivers/modules/pxe.py:282
msgid "iPXE boot is enabled but no HTTP URL or HTTP root was specified."
msgstr ""

#: ironic/drivers/modules/pxe.py:290
#, python-format
msgid ""
"Conflict: iPXE is enabled, but cannot be used with node%(node_uuid)s "
"configured to use UEFI boot"
msgstr ""

#: ironic/drivers/modules/pxe.py:444
#, python-format
msgid "Unsupported method (%s) passed to PXE driver."
msgstr ""

#: ironic/drivers/modules/seamicro.py:66
msgid "API endpoint. Required."
msgstr ""

#: ironic/drivers/modules/seamicro.py:68
msgid "server ID. Required."
msgstr ""

#: ironic/drivers/modules/seamicro.py:72
msgid "version of SeaMicro API client; default is 2. Optional."
msgstr ""

#: ironic/drivers/modules/seamicro.py:94
#, python-format
msgid "Invalid 'seamicro_api_version' parameter. Reason: %s."
msgstr ""

#: ironic/drivers/modules/seamicro.py:109
#, python-format
msgid "SeaMicro driver requires the following to be set: %s."
msgstr ""

#: ironic/drivers/modules/seamicro.py:308
msgid "Invalid volume id specified"
msgstr ""

#: ironic/drivers/modules/seamicro.py:324
msgid "No storage pools found for ironic"
msgstr ""

#: ironic/drivers/modules/seamicro.py:388
msgid "set_power_state called with invalid power state."
msgstr ""

#: ironic/drivers/modules/seamicro.py:420
#, python-format
msgid "Unsupported method (%s) passed to SeaMicro driver."
msgstr ""

#: ironic/drivers/modules/seamicro.py:439
msgid "No vlan id provided"
msgstr ""

#: ironic/drivers/modules/seamicro.py:477
msgid "No volume size provided for creating volume"
msgstr ""

#: ironic/drivers/modules/snmp.py:73
msgid "PDU manufacturer driver.  Required."
msgstr ""

#: ironic/drivers/modules/snmp.py:74
msgid "PDU IPv4 address or hostname.  Required."
msgstr ""

#: ironic/drivers/modules/snmp.py:75
msgid "PDU power outlet index (1-based).  Required."
msgstr ""

#: ironic/drivers/modules/snmp.py:78
#, python-format
msgid "SNMP protocol version: %(v1)s, %(v2c)s, %(v3)s  (optional, default %(v1)s)"
msgstr ""

#: ironic/drivers/modules/snmp.py:81
#, python-format
msgid "SNMP port, default %(port)d"
msgstr ""

#: ironic/drivers/modules/snmp.py:82
#, python-format
msgid "SNMP community.  Required for versions %(v1)s, %(v2c)s"
msgstr ""

#: ironic/drivers/modules/snmp.py:85
#, python-format
msgid "SNMP security name.  Required for version %(v3)s"
msgstr ""

#: ironic/drivers/modules/snmp.py:522
#, python-format
msgid "SNMP driver requires the following to be set: %s."
msgstr ""

#: ironic/drivers/modules/snmp.py:531
#, python-format
msgid "SNMPPowerDriver: unknown driver: '%s'"
msgstr ""

#: ironic/drivers/modules/snmp.py:537
#, python-format
msgid "SNMPPowerDriver: unknown SNMP version: '%s'"
msgstr ""

#: ironic/drivers/modules/snmp.py:546
#, python-format
msgid "SNMPPowerDriver: SNMP UDP port must be numeric: %s"
msgstr ""

#: ironic/drivers/modules/snmp.py:549
#, python-format
msgid "SNMPPowerDriver: SNMP UDP port out of range: %d"
msgstr ""

#: ironic/drivers/modules/snmp.py:556
#, python-format
msgid "SNMP driver requires snmp_community to be set for version %s."
msgstr ""

#: ironic/drivers/modules/snmp.py:562
#, python-format
msgid "SNMP driver requires snmp_security to be set for version %s."
msgstr ""

#: ironic/drivers/modules/ssh.py:58
msgid "IP address or hostname of the node to ssh into. Required."
msgstr ""

#: ironic/drivers/modules/ssh.py:60
msgid "username to authenticate as. Required."
msgstr ""

#: ironic/drivers/modules/ssh.py:61
msgid ""
"virtualization software to use; one of vbox, virsh, vmware, parallels. "
"Required."
msgstr ""

#: ironic/drivers/modules/ssh.py:65
msgid ""
"private key(s). One of this, ssh_key_filename, or ssh_password must be "
"specified."
msgstr ""

#: ironic/drivers/modules/ssh.py:67
msgid ""
"(list of) filename(s) of optional private key(s) for authentication. One "
"of this, ssh_key_contents, or ssh_password must be specified."
msgstr ""

#: ironic/drivers/modules/ssh.py:70
msgid ""
"password to use for authentication or for unlocking a private key. One of"
" this, ssh_key_contents, or ssh_key_filename must be specified."
msgstr ""

#: ironic/drivers/modules/ssh.py:73
msgid "port on the node to connect to; default is 22. Optional."
msgstr ""

#: ironic/drivers/modules/ssh.py:105
#, python-format
msgid "SSHPowerDriver '%(virt_type)s' is not a valid virt_type."
msgstr ""

#: ironic/drivers/modules/ssh.py:196
#, python-format
msgid "SSHPowerDriver '%(virt_type)s' is not a valid virt_type, "
msgstr ""

#: ironic/drivers/modules/ssh.py:287
#, python-format
msgid "SSHPowerDriver requires the following to be set: %s."
msgstr ""

#: ironic/drivers/modules/ssh.py:297
msgid "SSHPowerDriver requires ssh_port to be integer value"
msgstr ""

#: ironic/drivers/modules/ssh.py:318
msgid ""
"SSHPowerDriver requires one and only one of password, key_contents and "
"key_filename to be set."
msgstr ""

#: ironic/drivers/modules/ssh.py:327
#, python-format
msgid "SSH key file %s not found."
msgstr ""

#: ironic/drivers/modules/ssh.py:502
#, python-format
msgid "SSH connection cannot be established: %s"
msgstr ""

#: ironic/drivers/modules/drac/common.py:27
msgid "IP address or hostname of the DRAC card. Required."
msgstr ""

#: ironic/drivers/modules/drac/common.py:28
msgid "username used for authentication. Required."
msgstr ""

#: ironic/drivers/modules/drac/common.py:29
msgid "password used for authentication. Required."
msgstr ""

#: ironic/drivers/modules/drac/common.py:32
msgid "port used for WS-Man endpoint; default is 443. Optional."
msgstr ""

#: ironic/drivers/modules/drac/common.py:33
msgid "path used for WS-Man endpoint; default is \"/wsman\". Optional."
msgstr ""

#: ironic/drivers/modules/drac/common.py:35
msgid ""
"protocol used for WS-Man endpoint; one of http, https; default is "
"\"https\". Optional."
msgstr ""

#: ironic/drivers/modules/drac/common.py:65
#, python-format
msgid "'%s' not supplied to DracDriver."
msgstr ""

#: ironic/drivers/modules/drac/common.py:67
#, python-format
msgid "'%s' contains non-ASCII symbol."
msgstr ""

#: ironic/drivers/modules/drac/common.py:75
msgid "'drac_path' contains non-ASCII symbol."
msgstr ""

#: ironic/drivers/modules/drac/common.py:81
msgid "'drac_protocol' contains non-ASCII symbol."
msgstr ""

#: ironic/drivers/modules/drac/common.py:86
msgid "'drac_port' is not an integer value."
msgstr ""

#: ironic/drivers/modules/drac/common.py:89
#: ironic/drivers/modules/ilo/common.py:117
#, python-format
msgid ""
"The following errors were encountered while parsing driver_info:\n"
"%s"
msgstr ""

#: ironic/drivers/modules/drac/management.py:170
#, python-format
msgid ""
"Another job with ID \"%s\" is already created to configure the BIOS. Wait"
" until existing job is completed or is cancelled"
msgstr ""

#: ironic/drivers/modules/ilo/common.py:62
msgid "IP address or hostname of the iLO. Required."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:63
msgid "username for the iLO with administrator privileges. Required."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:65
msgid "password for ilo_username. Required."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:68
msgid "port to be used for iLO operations. Optional."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:69
msgid "timeout (in seconds) for iLO operations. Optional."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:101
#, python-format
msgid "'%s' not supplied to IloDriver."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:103
#, python-format
msgid ""
"The following parameters were mising while parsing driver_info:\n"
"%s"
msgstr ""

#: ironic/drivers/modules/ilo/common.py:112
#, python-format
msgid "'%s' is not an integer."
msgstr ""

#: ironic/drivers/modules/ilo/common.py:165
msgid "iLO license check"
msgstr ""

#: ironic/drivers/modules/ilo/common.py:255
#, python-format
msgid "Inserting virtual media %s"
msgstr ""

#: ironic/drivers/modules/ilo/common.py:278
#, python-format
msgid "Setting %s as boot device"
msgstr ""

#: ironic/drivers/modules/ilo/common.py:309
#, python-format
msgid "Setting %s as boot mode"
msgstr ""

#: ironic/drivers/modules/ilo/deploy.py:46
msgid "UUID (from Glance) of the deployment ISO. Required."
msgstr ""

#: ironic/drivers/modules/ilo/deploy.py:188
msgid "Error validating iLO virtual media deploy"
msgstr ""

#: ironic/drivers/modules/ilo/deploy.py:518
#, python-format
msgid "Unsupported method (%s) passed to iLO driver."
msgstr ""

#: ironic/drivers/modules/ilo/power.py:86
msgid "iLO get_power_status"
msgstr ""

#: ironic/drivers/modules/ilo/power.py:150
#, python-format
msgid "_set_power_state called with invalid power state '%s'"
msgstr ""

#: ironic/drivers/modules/ilo/power.py:159
msgid "iLO set_power_state"
msgstr ""

#: ironic/objects/base.py:161
msgid "Invalid version string"
msgstr ""

#: ironic/objects/base.py:349
#, python-format
msgid "Cannot load '%(attrname)s' in the base class"
msgstr ""

#: ironic/objects/base.py:358
msgid "Cannot save anything in the base class"
msgstr ""

#: ironic/objects/base.py:390
#, python-format
msgid "%(objname)s object has no attribute '%(attrname)s'"
msgstr ""

#: ironic/objects/base.py:439
#, python-format
msgid "'%(objclass)s' object has no attribute '%(attrname)s'"
msgstr ""

#: ironic/objects/conductor.py:56
msgid "Cannot update a conductor record directly."
msgstr ""

#: ironic/objects/utils.py:40
msgid "A datetime.datetime is required here"
msgstr ""

#: ironic/objects/utils.py:105
#, python-format
msgid "An object of class %s is required here"
msgstr ""

#: ironic/openstack/common/cliutils.py:271
#, python-format
msgid "No %(name)s with a name or ID of '%(name_or_id)s' exists."
msgstr ""

#: ironic/openstack/common/cliutils.py:279
#, python-format
msgid ""
"Multiple %(name)s matches found for '%(name_or_id)s', use an ID to be "
"more specific."
msgstr ""

#: ironic/openstack/common/gettextutils.py:301
msgid "Message objects do not support addition."
msgstr ""

#: ironic/openstack/common/gettextutils.py:311
msgid ""
"Message objects do not support str() because they may contain non-ascii "
"characters. Please use unicode() or translate() instead."
msgstr ""

#: ironic/openstack/common/imageutils.py:75
#, python-format
msgid "Invalid input value \"%s\"."
msgstr ""

#: ironic/openstack/common/imageutils.py:104
msgid "Snapshot list encountered but no header found!"
msgstr ""

#: ironic/openstack/common/lockutils.py:101
#, python-format
msgid "Unable to acquire lock on `%(filename)s` due to %(exception)s"
msgstr ""

#: ironic/openstack/common/log.py:287
#, python-format
msgid "Deprecated: %s"
msgstr ""

#: ironic/openstack/common/log.py:396
#, python-format
msgid "Error loading logging config %(log_config)s: %(err_msg)s"
msgstr ""

#: ironic/openstack/common/log.py:457
#, python-format
msgid "syslog facility must be one of: %s"
msgstr ""

#: ironic/openstack/common/log.py:700
#, python-format
msgid "Fatal call to deprecated config: %(msg)s"
msgstr ""

#: ironic/openstack/common/periodic_task.py:40
#, python-format
msgid "Unexpected argument for periodic task creation: %(arg)s."
msgstr ""

#: ironic/openstack/common/policy.py:395
#, python-format
msgid "Failed to understand rule %(rule)s"
msgstr ""

#: ironic/openstack/common/policy.py:405
#, python-format
msgid "No handler for matches of kind %s"
msgstr ""

#: ironic/openstack/common/policy.py:680
#, python-format
msgid "Failed to understand rule %(rule)r"
msgstr ""

#: ironic/openstack/common/processutils.py:59
msgid "Unexpected error while running command."
msgstr ""

#: ironic/openstack/common/processutils.py:62
#, python-format
msgid ""
"%(description)s\n"
"Command: %(cmd)s\n"
"Exit code: %(exit_code)s\n"
"Stdout: %(stdout)r\n"
"Stderr: %(stderr)r"
msgstr ""

#: ironic/openstack/common/processutils.py:143
#, python-format
msgid "Got unknown keyword args: %r"
msgstr ""

#: ironic/openstack/common/processutils.py:148
msgid "Command requested root, but did not specify a root helper."
msgstr ""

#: ironic/openstack/common/processutils.py:245
msgid "Environment not supported over SSH"
msgstr ""

#: ironic/openstack/common/processutils.py:249
msgid "process_input not supported over SSH"
msgstr ""

#: ironic/openstack/common/strutils.py:114
#, python-format
msgid "Unrecognized value '%(val)s', acceptable values are: %(acceptable)s"
msgstr ""

#: ironic/openstack/common/strutils.py:219
#, python-format
msgid "Invalid unit system: \"%s\""
msgstr ""

#: ironic/openstack/common/strutils.py:228
#, python-format
msgid "Invalid string format: %s"
msgstr ""

#: ironic/openstack/common/versionutils.py:88
#, python-format
msgid ""
"%(what)s is deprecated as of %(as_of)s in favor of %(in_favor_of)s and "
"may be removed in %(remove_in)s."
msgstr ""

#: ironic/openstack/common/versionutils.py:92
#, python-format
msgid ""
"%(what)s is deprecated as of %(as_of)s and may be removed in "
"%(remove_in)s. It will not be superseded."
msgstr ""

#: ironic/openstack/common/versionutils.py:96
#, python-format
msgid "%(what)s is deprecated as of %(as_of)s in favor of %(in_favor_of)s."
msgstr ""

#: ironic/openstack/common/versionutils.py:99
#, python-format
msgid "%(what)s is deprecated as of %(as_of)s. It will not be superseded."
msgstr ""

#: ironic/openstack/common/apiclient/base.py:224
#: ironic/openstack/common/apiclient/base.py:381
#, python-format
msgid "No %(name)s matching %(args)s."
msgstr ""

#: ironic/openstack/common/apiclient/client.py:233
msgid "Cannot find endpoint or token for request"
msgstr ""

#: ironic/openstack/common/apiclient/client.py:356
#, python-format
msgid ""
"Invalid %(api_name)s client version '%(version)s'. Must be one of: "
"%(version_map)s"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:41
#, python-format
msgid "Missing arguments: %s"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:74
#, python-format
msgid "Authentication failed. Missing options: %s"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:83
#, python-format
msgid "AuthSystemNotFound: %s"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:106
#, python-format
msgid "AmbiguousEndpoints: %s"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:114
msgid "HTTP Error"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:134
msgid "HTTP Redirection"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:142
msgid "HTTP Client Error"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:151
msgid "HTTP Server Error"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:161
msgid "Multiple Choices"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:170
msgid "Bad Request"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:180
msgid "Unauthorized"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:189
msgid "Payment Required"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:199
msgid "Forbidden"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:209
msgid "Not Found"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:219
msgid "Method Not Allowed"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:229
msgid "Not Acceptable"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:238
msgid "Proxy Authentication Required"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:247
msgid "Request Timeout"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:257
msgid "Conflict"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:267
msgid "Gone"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:277
msgid "Length Required"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:287
msgid "Precondition Failed"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:296
msgid "Request Entity Too Large"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:313
msgid "Request-URI Too Long"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:323
msgid "Unsupported Media Type"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:333
msgid "Requested Range Not Satisfiable"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:342
msgid "Expectation Failed"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:352
msgid "Unprocessable Entity"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:361
msgid "Internal Server Error"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:372
msgid "Not Implemented"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:382
msgid "Bad Gateway"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:391
msgid "Service Unavailable"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:401
msgid "Gateway Timeout"
msgstr ""

#: ironic/openstack/common/apiclient/exceptions.py:410
msgid "HTTP Version Not Supported"
msgstr ""

